gnomad_qc/v4/resources/basics.py

```diff
@@ -279,6 +283,11 @@ def _remove_ukb_dup_by_index(
     if split:
         logger.info("Splitting multiallelics...")
         vmt = vds.variant_data
+        if remove_extreme_multi_allelic:
```

Comment:
Let's just remove it no matter what, even if split is not True. Maybe switch to use filter_intervals with keep = False